src/core.js

```diff
@@ -1214,6 +1214,10 @@ define([
         this.data.core.form = form = parser.parseXForm(
             formXML, options, this, _this.data.core.parseWarnings);
         form.formName = this.opts().core.formName || form.formName;
+        if (formXML) {
+            _this._resetMessages(_this.data.core.form.errors);
+            _this._populateTree();
+        }
```

Comment:
Populate tree before binding form event handlers so mug property changes in `handleMugParseFinish` do not trigger a save button state change.